Automattic/jetpack
# My Plan: Free upgrade tracking

Pass the correct params to the `<UpgradeLink>` component for tracking to work

https://github.com/Automattic/jetpack/blob/master/_inc/client/components/upgrade-link/index.jsx#L27-L37

Fixes #12791 

#### Testing instructions:






* Go to wp-admin/admin.php?page=jetpack#/my-plan on a free plan
* Check that no warning or errors are thrown and that events are tracked

#### Proposed changelog entry for your changes:

* None
